include/nccl_ofi_rdma.h

```diff
@@ -565,6 +567,7 @@ typedef struct nccl_net_ofi_rdma_recv_comm_rail {
 typedef struct nccl_net_ofi_rdma_flush_buffer {
 	void *host_buffer;
 	size_t size;
+	size_t cpu_cache_line_size;
```

Comment:
This should be in the plugin struct, not the flush buffer struct.  It's not like the answer is going to change over time.

Reply:
ack